build_info/libxo-tools.control

```diff
@@ -0,0 +1,9 @@
+Package: libxo-tools
+Version: @DEB_LIBXO_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Homepage: https://github.com/Juniper/libxo
+Section: Utilites
+Priority: optional
+Depends: libintl8, perl, libxo (= @DEB_LIBXO_V@)
```

Comment:
I'm stopping here, a classic Cameron moment

Reply:
True true.

Reply:
Spell it right then.